AGENTS.md

```
@@ -0,0 +1,2136 @@
+|--------|----------|
+| `FirstPage(text, id, ...)` | Первая страница |
+| `PrevPage(text, id, ...)` | Предыдущая |
+| `NextPage(text, id, ...)` | Следующая |
+| `LastPage(text, id, ...)` | Последняя |
+| `SwitchPage(text, id, page, ...)` | На конкретную страницу |
+| `NumberedPager(id, ...)` | Нумерованный pager |
+| `CurrentPage(...)` | Текущая страница (текст) |
+| `StubScroll(id, total_pages=10)` | Заглушка скролла |
+
+**Request-кнопки:**
+| Виджет | Описание |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)